### PR TITLE
Feature/initial_html_css.md

### PR DESCRIPTION
**Pull request by JeffKersting**
#### What does  this PR do?
-Initial framework and styling and application of framework created.
#### Where should the reviewer start?
-Open index.html file in web browser
#### How should this be manually tested?
-Alter screen sizes using dev tools
#### Any background context you want to provide?
-Most of the objects are made to be responsive

### CODE REVIEW

**Comment by DrewBradley** [comment]
The naming conventions in the HTML look very clear and understandable. However, we should change the title of the search bar to 'search-input' instead of 'search-idea".

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,184 @@
+body{
+  font-family: Open Sans, sans-serif;
+  background-color: #EAEAF4;
+}
+
+.sidebar{
```
Comment (colefiscus) [inline styles.css:6]: This is all looking good! I say let's make sure to check the style guide to see if spaces are needed in spots like this one.

**Review by colefiscus — APPROVED** [review]
Looks like we have a solid comp build-out! We can always go back and edit things if they start falling out of place.

File: index.html [hunk 2]
```diff
@@ -6,7 +6,34 @@
...
+      <h2>Filter Starred Ideas</h2>
+      <button type="button" class="show-favorite">Show Starred Ideas</button>
+    </section>
+    <section class="idea-form">
+      <label for="title-input">Title</label>
+      <input type="text" class="title-input" name="title-input">
+      <label for="body-input">Body</label>
+      <input type="text" class="body-input" name="body-input">
+      <button type="button" name="save-button" class="save-button">Save</button>
+      <input type="text" class="search-idea" placeholder="Search ideas">
```
Comment (DrewBradley) [inline index.html]: The class name of this element seems confusing. "searchIdeas" would be a great name for a function, but seems like a confusing name for an input element.

File: index.html [hunk 3]
```diff
@@ -6,7 +6,34 @@
...
+      <article class="idea-container">
+        <span class="favorite-delete">
+          <button type="button" name="button" class="star"></button>
+          <button type="button" name="button" class="delete"></button>
+        </span>
+        <p class="idea-title">Idea Title</p>
+        <p class="idea-body">Idea body. Dont ever play yourself. Every chance I get, I water the plants, lion!</p>
+        <span class="comment">
+          <button type="button" name="comment-button" class="comment-button"></button>
+          <label for="comment-button"></label>
```
Comment (DrewBradley) [inline index.html]: We need to make sure this label element has a clear class name so that it can be accessed in the DOM.

File: styles.css [hunk 4]
```diff
@@ -29,21 +28,73 @@ h2{
...
-
-.idea-grid{
+input {
+  border: 2px solid #5356A4;
+  border-radius: 2px;
+  margin: 1% 0% 2% 0%;
+  font-size: 1em;
+  padding: 2%;
+}
+.body-input {
```
Comment (JeffKersting) [inline styles.css:72]: In the future, we can add a shadow-box to this element.

**Pull request merged**